test/stacktraces.jl

```diff
@@ -0,0 +1,80 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# Tests for /base/stacktraces.jl
+
+# Some tests don't currently work for Appveyor 32-bit Windows
+const APPVEYOR_WIN32 = (
+    OS_NAME == :Windows && WORD_SIZE == 32 && get(ENV, "APPVEYOR", "False") == "True"
```

Comment:
i think it should be fixed now by https://github.com/JuliaLang/julia/commit/9ce59c858a47057cdab9890932eabd9fe8c62377#diff-d4b80bf89e46982070f234dd7cc344a9L1107

the wrong return value there (from a post-v0.4 code reorganization) sometimes causes backtraces to become truncated / corrupted